src/components/layout/section-content/index.tsx

```diff
@@ -8,7 +8,7 @@ export interface SectionContentProps {
 }
 
 const Container = styled.div<SectionContentProps>`
-  max-width: 1200px;
+  max-width: 1160px;
```

Comment:
![Screen Shot 2021-01-31 at 1 01 13 PM](https://user-images.githubusercontent.com/15198653/106383129-68ec7980-63c4-11eb-886c-bab2148daae4.png)
to znamena ze:
`containerWidth = 376*3 + 16*2 = 1160`